editor/block-mover/mover-label.js

```diff
@@ -0,0 +1,73 @@
+import { __, sprintf } from 'i18n';
+
+export function blockMoverLabel( selectedCount, { type, firstPosition, isFirst, isLast, dir } ) {
```

Comment:
It'd be good to add a function comment here describing purpose and spec. Also, maybe `getBlockMoverLabel` is a better name.